at_secondary/at_persistence_secondary_server/lib/src/config/at_config.dart

```diff
@@ -14,6 +12,7 @@ class AtConfig {
 
   ///stores 'Configuration' type under [configkey] in secondary.
   String configKey = 'configKey';
+  static const server = AtServerAnnotation();
```

Comment:
The line of code above is being added in many files. These `server` and `client` top-level constants should instead be exported by the package - see https://pub.dev/documentation/meta/latest/meta/visibleForTesting-constant.html for example in the dart `meta` package